packages/transit-vehicle-overlay/src/index.story.js

```diff
@@ -133,7 +135,7 @@ function rectangles(popup = true) {
   // give action to the popup vehicle tracking button
   VehiclePopup.defaultProps.setTracked = (vehicle, isTracked) => {
     updateTrackedVehicle(vehicle, isTracked);
-    if (!isTracked) recenter(null, 0, 0); // no longer tracking so clear out cached coords
+    if (!isTracked) recenter(null, 0, doTracking, 0); // no longer tracking so clear out cached coords
```

Comment:
What is the intent here? I am confused with why arguments are changing. According to https://github.com/opentripplanner/otp-ui/blob/master/packages/transit-vehicle-overlay/src/utils/recenter.js#L29, only 3 arguments are needed, the first being the map and the last two being coordinates. As written, untracking a vehicle seems to do nothing.

Reply:
Hey Binh

A customer can select a vehicle, which then highlights that vehicle, shows the pattern geometry of the vehicle, and optionally pan/zooms the map to that tracked vehicle.  Was asked to add a switch for the customer to control that last bit, allowing an on or off state of the map pan / zoom (of the selected vehicle) ... and when switched off, tracking performs the rest of the tracking functionality minus the map pan/zoom to the vehicle's position.

I originally hoped to switch map.panTo on/off by setting the TransitVehicleOverlay's onRecenterMap (PropType.func) prop to either null or a function that didn't have a map.panTo().  Leaflet throws a bunch of errors if I try to replace the onRecenterMap function on a mounted component ... so no switching back and forth to the panTo() no panTo().   

So my second option was to keep the one function prop, and pass a boolean into the prop, that when it is false, it will not execute the panTo.  

Does this explaination help?  If so, I'll add it to the comments in utils/recenter.js.

Reply:
BTW, I'm going to rename that doTracking variable to doPanMap to avoid confusion in the story.

Reply:
> So my second option was to not muck with the onRecenterMap() prop, but rather pass a boolean into onRecenterMap that, when false, will not execute the map.panTo.

Indeed that is how it looks like, and you probably took care of it already when setting up `recenterFlyTo` and `recenterPanTo` around line 111.

For this particular function call, if you step into that function call, however, [this code](https://github.com/opentripplanner/otp-ui/blob/master/packages/transit-vehicle-overlay/src/utils/recenter.js#L41) gets called, and the arguments passed don't match the arguments of that code, that's where it is confusing.

Reply:
So to clarify, the signature of the `recenter` function here and elsewhere is `recenter(map, lat, lon)`, so as written, this is happening:
- `map = null`
- `lat = 0`
- `lon = true/false?`

Reply:
Resolved with https://github.com/opentripplanner/otp-ui/pull/255/commits/ad9b2124727a8718a90d3cb3bec5c451f4348a4a (undoes the changes to invocations of `recenter`).